pkg/kev/services_test.go

```diff
@@ -48,8 +49,9 @@ var _ = Describe("ServiceConfig", func() {
 
 			It("success if the necessary labels are present", func() {
 				err := ServiceConfig{Labels: composego.Labels{
-					config.LabelWorkloadLivenessProbeType: kubernetes.ProbeTypeCommand.String(),
-					config.LabelWorkloadReplicas:          "1",
+					config.LabelWorkloadLivenessProbeType:    kubernetes.ProbeTypeCommand.String(),
```

Comment:
I guess a couple more specs covering other probe types (for both expected success and failure cases) would be good to have as well.

Reply:
this is very covered throughout but I can add some more specific to the schema validation

Reply:
Yeh, would be good to add. Thanks.

Reply:
added